sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/mongodb.yaml

```
@@ -144,13 +140,7 @@ provider:
         post:
           # Avoid matching values found on SourceGraph that look like dummy passwords or insertions.
           patternNot:
-            "^(db|my)?_?pass(word|wd)?$|\
-            ^\\*{3,}$|\
-            ^\\$[A-Z_]+$|\
-            ^\\$[{(]|\
-            ^\\{{2,}|\
-            ^<[a-z \\-]+>$|\
-            ^--[a-z\\-]+$"
+            - "^(db|my)?_?pass(word|wd)?$|^\\*{3,}$|^\\$[A-Z_]+$|^\\$[{(]|^\\{{2,}|^<[a-z \\-]+>$|^--[a-z\\-]+$"
```

```suggestion
            - "(?i)^(db|my)?_?pass(word|wd)?$"
            - "^\\*{3,}$"
            - "^\\$[A-Z_]+$"
            - "^\\$[{(]"
            - "^\\{{2,}"
            - "^<[a-z \\-]+>$
            - "^--[a-z\\-]+$"
```

I think I don't get why you merged back this one

---

You may not to recheck this again, as there were other changes in the meantime which changed the patterns